<launch>


    <rosparam command="load" file="$(find data_preprocessing)/config/tower.yaml" />

    <node pkg="data_preprocessing" type="data_preprocessing" name="data_preprocessing" output="screen" />


</launch>

